.github/workflows/vanilla_ubuntu.yml

```diff
@@ -0,0 +1,148 @@
+---
+name: Vanilla Ubuntu
+
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+
+jobs:
+
+  lint-test:
+    name: Lint
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out the codebase.
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3.7.
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+
+      - name: Install test dependencies.
+        run: pip3 install yamllint ansible-lint ansible
+
+      - name: Version check
+        run: |
+          ansible --version
+          ansible-lint --version
+          yamllint --version
+
+      - name: Run yamllint.
+        run: yamllint .
+
+      - name: Run ansible-lint.
```

Comment:
@dgvigil - There seems to be a fair number of errors when running ansible-lint...not sure if it's pickup the proper setting within `.ansible-lint` file to ignore handler errors...again - could just be another issue to create separately

Reply:
This will be addressed in a future PR.  The warnings will go away with a line by line `# noqa ` to tell ansible-lint to ignore these.